src/GeneralStateTestsFiller/stCreate2/CREATE2_HighNonceDelegatecallFiller.yml

```diff
@@ -0,0 +1,588 @@
+CREATE2_HighNonceDelegatecall:
+  _info:
+    comment: "Delegate calls CREATE/CREATE2 from an account with max allowed nonce/max allowed nonce - 1."
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+    000000000000000000000000fffffffffffffffe:
+      balance: 0
+      code: |
+        :yul
+        {
+          let createtype    := calldataload(0)
+          let contextnonce  := sload(0xffff)
+
+          // initcode: { return(0, 1) }
```

Comment:
Can also track the existance of created contracts.
If init code puts smth to storage you will see the address in post state and would be able to check the storage

Reply:
Added sstore(1, 1) to created contracts' code, call all contracts after creation, and verify the storage in each contract.